src/components/admin/HearingFormStep2.jsx

```diff
@@ -1,13 +1,11 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { injectIntl, intlShape, FormattedMessage } from 'react-intl';
 import { v1 as uuid } from 'uuid';
 import { head, last } from 'lodash';
-import Accordion from 'react-bootstrap/lib/Accordion';
-import { Button } from 'hds-react';
+import { Accordion, Button } from 'hds-react';
 import ButtonToolbar from 'react-bootstrap/lib/ButtonToolbar';
```

Comment:
This is the only react-bootstrap component left in the HearingForm components so I guess this could be removed in this PR aswell?

Reply:
Removed: [a6c3fca](https://github.com/City-of-Helsinki/kerrokantasi-ui/pull/1083/commits/a6c3fca961eb1cd9ed346381de59e134209b6fa2)